playbooks/robusta_playbooks/oom_killer.py

```diff
@@ -31,6 +31,65 @@ class OomKillerEnricherParams(ActionParams):
 NODE_MEMORY_THRESHOLD = 0.95
 
 
+@action
+def pod_oom_killer_enricher(
+    event: PodEvent,
+):
+    pod = event.get_pod()
+
+    resource_requests = pod_requests(pod)
+    resource_limits = pod_limits(pod)
+
+    finding = Finding(
+        title=f"Pod {pod.metadata.name} in namespace {pod.metadata.namespace} OOMKilled results",
+        aggregation_key="pod_oom_killer_enricher",
+        severity=FindingSeverity.HIGH
+    )
+    logs_blocks: List[BaseBlock] = []
+    oom_killed_status = None
+    for container_status in pod.status.containerStatuses:
+        if container_status.state.terminated and "OOMKilled" in container_status.state.terminated.reason:
+            oom_killed_status = container_status.state.terminated
+        elif container_status.lastState.terminated and "OOMKilled" in container_status.lastState.terminated.reason:
+            oom_killed_status = container_status.lastState.terminated
+
+    labels = [("Pod", pod.metadata.name), ("Namespace", pod.metadata.namespace),
+              ("Memory limit", "None" if not resource_limits.memory else f"{resource_limits.memory}MB"),
+              ("Memory requests", "None" if not resource_requests.memory else f"{resource_requests.memory}MB"),
+              ("Node Name", pod.spec.nodeName),
+              ]
+    node: Node = Node.readNode(pod.spec.nodeName).obj
+    if node:
+        logging.warning(
+            f"Node {pod.spec.nodeName} not found for OOMKilled pod {pod.metadata.name}"
+        )
+        allocatable_memory = PodResources.parse_mem(node.status.allocatable.get("memory", "0Mi"))
+        capacity_memory = PodResources.parse_mem(node.status.capacity.get("memory", "0Mi"))
+        node_labels = [("Node memory allocatable (free)", f"{allocatable_memory}MB"),
```

Comment:
I would remove the "(free)" 
Allocatable isn't free. It's capacity - reserved for k8s